### PR TITLE
fix(apiMulti): access api multi over nuxt instance in pinia store

### PR DESCRIPTION
**Pull request by gaucj-t**
replacec vue instance with nuxt instance from within pinia stores

### CODE REVIEW

File: store/auth.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ const initState = (): AuthState => {
 }
 
 export const useAuthStore = defineStore('auth', () => {
-    const { proxy: root } = getCurrentInstance() || {}
+    const store = useAuthStore()
```
Comment (PeterThrels) [inline store/auth.ts:30]: the store calls itself?

**Review by PeterThrels — APPROVED** [review]

File: store/auth.ts (hunk 1 above)
Comment (gaucj-t) [reply store/auth.ts:30]: yea we can access the context of the store and access nuxt context

Comment (oleggrech-t) [reply store/auth.ts:30]: Would this be available for all stores? i.e. accessing the multi api

**Review by bartolo-maureen — APPROVED** [review]

File: plugins/weavr/types/index.d.ts [hunk 2]
```diff
@@ -4,26 +4,18 @@ import '@nuxt/types'
 declare module 'vue/types/vue' {
     interface Vue {
         $weavrComponents: any
-        $weavrSetUserToken: (token) => {}
+        $weavrSetUserToken: (token: any) => {}
```
[the remark this answers is not in the document]
Comment (gaucj-t) [reply plugins/weavr/types/index.d.ts]: sorted

**Pull request merged**